docs/src/replication.md

```diff
@@ -297,29 +294,29 @@ spec:
     size: 1Gi
 ```
 
-Replication slots must be carefully monitored in your infrastructure. By default,
+You must carefully monitor replication slots in your infrastructure. By default,
 we provide the `pg_replication_slots` metric in our Prometheus exporter with
-key information such as the name of the slot, the type, whether it is active,
+key information such as the name of the slot, the type, whether it's active, and
 the lag from the primary.
 
 !!! Seealso "Monitoring"
-    Please refer to the ["Monitoring" section](monitoring.md) for details on
+    See [Monitoring](monitoring.md) for details on
     how to monitor a CloudNativePG deployment.
 
 ### Capping the WAL size retained for replication slots
 
 When replication slots is enabled, you might end up running out of disk
 space due to PostgreSQL trying to retain WAL files requested by a replication
-slot. This might happen due to a standby that is (temporarily?) down, or
+slot. This might happen due to a standby that's (temporarily) down,
```

Comment:
Not sure why there was a question mark after temporarily. If there is more to be said about the state, use a separate sentence.